README.md

```diff
@@ -1 +1,80 @@
-# terraform-google-utils
\ No newline at end of file
+# terraform-google-utils
+
+This module provides a way to get the shortnames for given GCP region(s). For example, "us-central1" becomes "usc1".
+
+This module does not communicate with GCP in any way.
+
+## Usage
+
+Basic usage of this module is as follows:
+
+```hcl
+module "utils" {
+  source  = "terraform-google-modules/utils/google"
+  version = "~> 0.1"
+  regions = ["us-central1", "us-west1"]
+}
+
+output "region_short_name_map" {
```

Comment:
Just outputting the output isn't a very useful example. We should do something like setting a local to `module.utils. region_short_name_map["us-central1"]`.